.github/workflows/e2e-tests.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.17'
+      - run: mkdir -p artifacts
+      - run: go install github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
Had to install ginkgo v2 cli. Calling via go run github.com/onsi/ginkgo/ginkgo still runs v1.

Reply:
I think I found a way to execute it similar to before. I had to remove -mod=vendor though